x/nexus/client/cli/tx.go

```diff
@@ -80,3 +81,29 @@ func GetCmdDeregisterChainMaintainer() *cobra.Command {
 	flags.AddTxFlagsToCmd(cmd)
 	return cmd
 }
+
+// GetCmdActivateChain returns the cli command to activate the given chains
+func GetCmdActivateChain() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "activate-chain [[chains]...]",
```

Comment:
it unclear to me from this usage string how multiple chains would have to be formatted

Reply:
changed to `activate-chain [chains]...`